src/extension/windows/wpfPlatform.ts

```diff
@@ -24,59 +23,58 @@ export class WpfPlatform extends WindowsPlatform {
     }
 
     public runApp(enableDebug: boolean = true): Q.Promise<void> {
-        const extProps = {
+        let extProps = {
             platform: {
                 value: "wpf",
                 isPii: false,
             },
         };
 
+        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersion, extProps);
+
         return TelemetryHelper.generate("WpfPlatform.runApp", extProps, () => {
             const env = this.getEnvArgument();
 
             if (enableDebug) {
                 this.runArguments.push("--proxy");
             }
 
-            return ReactNativeProjectHelper.getReactNativeVersion(this.runOptions.projectRoot)
-                .then(version => {
-                    if (!semver.gt(version, WpfPlatform.WPF_SUPPORTED)) {
-                        throw new Error(localize("DebuggingWPFPlatformIsNotSupportedForThisRNWinVersion", "Debugging WPF platform is not supported for this react-native-windows version({0})", version));
-                    }
+            if (!semver.gt(this.runOptions.reactNativeVersion, WpfPlatform.WPF_SUPPORTED)) {
```

Comment:
Why we change that?

Reply:
I suppose that we don't need to get React Native version here anymore, because we already have a version in `this.runOptions`.